code/obj/item/gun/ammo.dm

```diff
@@ -688,6 +718,7 @@
 /obj/item/ammo/bullets/abg
 	sname = "12ga Rubber Slug"
 	name = "12ga rubber slugs"
+	desc = "A box of rubber slugs. Despite being nonlethal, these things can still seriously maim."
```

Comment:
'maim' seems harsh for these, considering they do relatively little damage and are mostly for knockback.
Maybe 'these things can still knock you across the room' or something.